### PR TITLE
FizzBuzz- scala

### PR DESCRIPTION
**Pull request by sukeshni**
@shunjikonishi Please review

### CODE REVIEW

File: src/test/scala/fizzbuzztest.scala [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import org.scalatest._
+import FizzBuzz._
+
+class TestFizzBuzzSpec extends FlatSpec with Matchers {
+
+  "fizzBuzz" should " return corresponding 'FizzBuzz'values for number passed" in {
+
+    val data = new Array[String](101)
```
Comment (shunjikonishi) [inline src/test/scala/fizzbuzztest.scala]: Why you declared array here?
You can use block scope variable in for block.

**Comment by shunjikonishi** [comment]
@sukeshni fix me

**Comment by sukeshni** [comment]
@shunjikonishi Fixed your point. Please review.

**Comment by shunjikonishi** [comment]
@sukeshni fix me
Using `var` is not good in Scala.
You can rewrite it with `val`.

**Comment by sukeshni** [comment]
@shunjikonishi Please Review

**Comment by shunjikonishi** [comment]
LGTM

**Pull request closed without merging**